Source/Core/Core/PowerPC/Jit64/JitAsm.cpp

```diff
@@ -81,6 +81,7 @@ void Jit64AsmRoutineManager::Generate()
   ResetStack(*this);
 
   SUB(32, PPCSTATE(downcount), R(RSCRATCH2));
+  NOP();  // Prevent op-Jcc fusion.
```

Comment:
Why are we preventing op-Jcc fusion? I though the emitter now handles this.

Reply:
Here, the later JLE in `dispatcher` will be detected as fusable with the SUB, and thus the SUB will be memmoved. However, this invalidates the value in `dispatcher`.

An alternative way of preventing this is to modify `GetCodePtr` as so:

```
const u8* XEmitter::GetCodePtr() const
{
  jcc_fusion_type = JccFusionType::None;
  return code;
}
```

i.e. prevent the possibility of `memmove`-ing code across such locations

Reply:
It's fine, but would be nice if the comment elaborated.

Reply:
Done.